packages/telemetry/src/system-info.ts

```diff
@@ -39,18 +42,21 @@ export type SystemInfo = {
 	isWSL: boolean;
 	isCI: boolean;
 	ciName: string | null;
-	astroVersion: string;
 };
 
 let meta: SystemInfo | undefined;
 
-export function getSystemInfo(astroVersion: string): SystemInfo {
+export function getSystemInfo(versions: {viteVersion: string, astroVersion: string}): SystemInfo {
 	if (meta) {
 		return meta;
 	}
 
 	const cpus = os.cpus() || [];
 	meta = {
+		// Version information
+		nodeVersion: process.version.replace(/^v?/, ''),
+		viteVersion: versions.viteVersion,
+		astroVersion: versions.astroVersion,
```

Comment:
Nice! This will be great to have.

Reply:
This was actually already being passed as a property of the individual session event. Now it is passed as system metadata so that every event doesn't need to re-include it as a property of the function call.